ProcessLib/HeatTransportBHE/LocalAssemblers/HeatTransportBHELocalAssemblerSoil-impl.h

```diff
@@ -61,8 +62,10 @@ HeatTransportBHELocalAssemblerSoil<ShapeFunction, IntegrationMethod>::
         auto const& sm = _shape_matrices[ip];
         double const w = _integration_method.getWeightedPoint(ip).getWeight() *
                          sm.integralMeasure * sm.detJ;
-        _ip_data.push_back(
-            {sm.N.transpose() * sm.N * w, sm.dNdx.transpose() * sm.dNdx * w});
+        _ip_data.emplace_back(
+            sm.N, sm.dNdx,
+            _integration_method.getWeightedPoint(ip).getWeight() *
+                sm.integralMeasure * sm.detJ);
```

Comment:
The third argument is identical to the variable `w` above. You could write:
```
_ip_data.emplace_back(sm.N, sm.dNdx, w);
```

Reply:
You are right. Changed.